### PR TITLE
Added inventory to deploy an acme controller

### PR DESCRIPTION
**Pull request by sabre1041**
Inventory to deploy an acme controller leveraging the `openshift-applier`

### CODE REVIEW

File: files/acme/deployment/template.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        - name: "${NAME}"
+          image: "${IMAGE_NAME}:${IMAGE_VERSION}"
+          imagePullPolicy: IfNotPresent
+          ports:
+          - containerPort: 80
+          env:
+          - name: OPENSHIFT_ACME_ACMEURL
+            value: "${OPENSHIFT_ACME_ACMEURL}"
+          - name: OPENSHIFT_ACME_LOGLEVEL
+            value: "${OPENSHIFT_ACME_ACMEURL}"
```
Comment (pabrahamsson) [inline files/acme/deployment/template.yml]: Should this be OPENSHIFT_ACME_LOGLEVEL?

File: files/acme/deployment/template.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+- description: Version of the Let's Encrypt Image
+  name: IMAGE_VERSION
+  value: latest
+  required: true
+- description: URL of the Let's Encrypt Endpoint
+  name: OPENSHIFT_ACME_ACMEURL
+  value: https://acme-v01.api.letsencrypt.org/directory
+  required: true
+- description: Application Log Level
+  name: OPENSHIFT_ACME_ACMEURL
```
Comment (pabrahamsson) [inline files/acme/deployment/template.yml]: Same here, OPENSHIFT_ACME_LOGLEVEL?

**Comment by sabre1041** [comment]
@pabrahamsson thanks for finding those issues with the template. They have been corrected.

**Comment by etsauer** [comment]
@pabrahamsson @sabre1041 is this good to merge?

**Comment by etsauer** [comment]
@sabre1041 could you add "menu items" to the main readme to point to the cakephp and acme stuff?

**Comment by sabre1041** [comment]
@etsauer updated the README

**Comment by pabrahamsson** [comment]
@etsauer I haven't had a chance to test this out yet and I'm also not terribly familiar with Let's Encrypt. Do we need testing or are my two :eyes: enough? If yes then LGTM.

**Comment by oybed** [comment]
This is something that’s very timely. I’ll add it to my list of items to test right away as we need this for Labs ASAP. Alternatively; just merge and we can submit any updates in follow-up PRs.

**Comment by etsauer** [comment]
@sabre1041 @pabrahamsson @oybed anyone tested this yet?

File: files/acme/policy/clusterrolebinding.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ClusterRoleBinding
+metadata:
+  name: acme-controller
+roleRef:
+  name: acme-controller
+subjects:
+- kind: ServiceAccount
+  name: acme
+  namespace: acme
```
Comment (oybed) [inline files/acme/policy/clusterrolebinding.yml]: this locks it in on always being deployed as `acme` ... while the above template allows for setting the namespace to something else than `acme`. Can we change this file to a template as well to allow for a custom name?

File: files/acme/project/project.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+kind: ProjectRequest
+apiVersion: v1
+metadata:
+  name: acme
```
Comment (oybed) [inline files/acme/project/project.yml]: Same comment above - probably should use a template to allow for a custom name of the namespace

Comment (oybed) [reply files/acme/project/project.yml]: With the resent merge (thank you), you can now just reference the generic projectrequest template:
https://raw.githubusercontent.com/redhat-cop/cluster-lifecycle/master/files/projectrequest/template.yml
... and supply a params file with these values

**Comment by sabre1041** [comment]
@oybed Addressed your feedback. Please review

**Comment by oybed** [comment]
LGTM - merging

**Pull request merged**